scripts/diffStorageLayout.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+CONTRACT=$1
+if [ -z "$CONTRACT" ]
+then
+  echo "Error: CONTRACT should be name of contract that you want to inspect storage layout of. e.g. Arbitrum_SpokePool"
+  exit 1
+fi
+
+echo "Creating a storage layout snapshot of the $CONTRACT contract at ./storage-layouts/proposed.$CONTRACT.json"
+echo "⏳ This can take a few minutes to complete if you haven't run a forge command recently"
+touch ./storage-layouts/proposed.$CONTRACT.json
```

Comment:
Hi @nicholaspai, you don't need to create a file here as you redirect the output from Foundry storage inspect to the file (line 14). If the inspect command fails, the empty file will be created anyway

Reply:
nice find, thanks!